draft-ietf-jsonpath-base.md

```diff
@@ -1521,29 +1527,43 @@ A function expression is well-typed if all of the following are true:
   parameter according to one of the following rules:
    * The argument is a function expression with declared result type that is the same as the declared type of the parameter.
    * The argument is a literal primitive value and the declared type of the parameter is `ValueType`.
+   * The argument is a Singular Path and the declared type of the parameter is `SingleNodeType`.
    * The argument is a Singular Path or `filter-path` (which includes
      Singular Paths) and the declared type of the parameter is `NodesType`.
 
-{{tab-exist-value}} summarizes how the two conversion functions can be
-used to make use of functions returning nodelists, using a
+{{tab-exist-value}} summarizes how the two conversion functions behave
```

Comment:
I'm not convinced of the value of the following two tables, but they may be helpful here to emphasise the effect of this PR.